<launch>
    <arg name="start_pos"        default="Hill1_SC" />
    <arg name="autonsystem_args" default="--controller stanley --dist 0.0 --traj buggycourse_safe.json --self_name SC" />
    <arg name="velocity"         default="15.0" />
    <arg name="buggy_name"       default="SC" />

    <node name="foxglove" pkg="foxglove_bridge" type="foxglove_bridge" />

    <arg name="manual_vel"          default="false" />
    <arg name="auto_vel"            default="false" />

    <!-- <node pkg="tf" type="static_transform_publisher" name="static_transform" args="0 0 0 0 0 0  /base_footprint /base_link  100"/> -->
    <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 0 0 0 1 /imu /base_footprint  10 " />

    <node name="ekf_translator" pkg="buggy" type="ekf.py" respawn="true" output="screen" />  

    <node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf">
        <param name="output_frame" value="odom_combined"/>
        <param name="freq" value="30000000.0"/>
        <param name="sensor_timeout" value="1.0"/>  
        <param name="odom_used" value="true"/>
        <param name="imu_used" value="true"/>
        <param name="vo_used" value="false"/>
        <param name="gps_used" value="false"/>
        <param name="debug" value="false"/>
        <param name="self_diagnose" value="false"/>

        <remap from="imu_data" to="/imu/data"/>
        <remap from="gps" to="/nav/odom"/>
        <remap from="odom" to="/nav/odom"/>
    </node>
</launch>